pom.xml

```diff
@@ -126,6 +128,11 @@
                 <artifactId>fest-assert</artifactId>
                 <version>${version.fest-assert}</version>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
```

Comment:
Then I would recommend removing `assertj` entirely and make a new PR introducing it. Otherwise this duality might stay like that forever.